ChunkyDocs/docs/plugins/plugin_development.md

```diff
@@ -84,13 +88,22 @@ public class DemoPlugin implements Plugin {
 !!! info "About the main method"
     The `main` method is added only for convenience. This way, you can launch Chunky with this plugin enabled directly from within your IDE, which is also useful for attaching a debugger. When loading a plugin from a jar, Chunky will create an instance of the plugin class and invoke the `attach` method. You should put all plugin initialization logic there.
 
+---
+
 ## Demo plugins
 
 To demonstrate some features of the Plugin API, llbit created a few demo plugins.
 
-- [Ambient Occlusion Plugin](https://github.com/llbit/Chunky-AOPlugin)
-- [Depth Buffer Plugin](https://github.com/llbit/Chunky-DepthPlugin)
-- [Block Plugin Template](https://github.com/llbit/Chunky-BlockMod)
-- [Tab Plugin Template](https://github.com/llbit/Chunky-TabMod)
+- <a href="https://github.com/llbit/Chunky-AOPlugin" target="_blank">Ambient Occlusion Plugin</a>
+
+- <a href="https://github.com/llbit/Chunky-DepthPlugin" target="_blank">Depth Buffer Plugin</a>
+
```

Comment:
Maybe consider replacing these with `ChunkyAOV` since they use the old, deprecated interface for setting renderers?

Reply:
Changed to:

To demonstrate some features of the Plugin API, llbit created a few demo plugins.

- <a href="https://github.com/llbit/Chunky-AOPlugin" target="_blank">Ambient Occlusion Plugin</a>

- <a href="https://github.com/llbit/Chunky-DepthPlugin" target="_blank">Depth Buffer Plugin</a>

- <a href="https://github.com/llbit/Chunky-BlockMod" target="_blank">Block Plugin Template</a>

- <a href="https://github.com/llbit/Chunky-TabMod" target="_blank">Tab Plugin Template</a>

The Ambient Occlusion plugin and the Depth Buffer plugin use a deprecated API to add renderers to Chunky. The Chunky AOV plugin adds these renderers using the new API.